pages/index.js

```diff
@@ -1,6 +1,6 @@
 import Link from 'next/link'
 import { useAuthContext } from '../src/auth/AuthContextProvider'
-import Footer from './Footer';
+import Footer from '../src/auth/Footer';
```

Comment:
move footer to `/src` and out of `auth` folder as it is not part of the authentication